src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -6,75 +6,67 @@
 
 namespace Stryker.Core.Initialisation
 {
-    public class ProjectFileReader
-    {
+    public class ProjectFileReader {
```

Comment:
This project follows the [Microsoft Coding conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#layout-conventions). They place brackets under the class and method names.

Reply:
Yeah, I noticed that on the second commit diff.. I am unsure what happened.  I can go through and fix it.  My VS does this even though C# is set up to place on new lines.  I have to fight it all the time, and it annoys me to no end.